lib/taskana-cdi/pom.xml

```diff
@@ -25,29 +25,30 @@
 			<version>${project.version}</version>
 		</dependency>
 
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-simple</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
can you please move this down a line (under TEST dependencies) ?

Reply:
fixed